pkgs/development/python-modules/connexion/default.nix

```
@@ -93,12 +99,13 @@ buildPythonPackage rec {
     "test_writeonly"
   ];
 
-  meta = with lib; {
+  meta = {
     description = "Swagger/OpenAPI First framework on top of Flask";
     homepage = "https://github.com/spec-first/connexion";
     changelog = "https://github.com/spec-first/connexion/releases/tag/${version}";
-    license = licenses.asl20;
-    maintainers = [ ];
+    license = lib.licenses.asl20;
+    broken = stdenv.hostPlatform.isDarwin;
```

```suggestion
    badPlatforms = [
      "aarch64-darwin"
      "x86_64-darwin"
    ];
```
I think that this is the recommended way as it gives a clearer message.
Also, could you add a comment stating why we mark it as broken ?